src/gc/gcee.cpp

```diff
@@ -390,6 +390,7 @@ size_t GCHeap::GetNow()
     return hp->get_time_now();
 }
 
+#if defined(GC_PROFILING) //UNIXTODO: Enable this for FEATURE_EVENT_TRACE
 void ProfScanRootsHelper(Object** ppObject, ScanContext *pSC, DWORD dwFlags)
 {
 #if defined(GC_PROFILING) || defined(FEATURE_EVENT_TRACE)
```

Comment:
Since GC_PROFILING is now at a higher scope, please remove it from here and update the corresponding #endif.